mne/viz/utils.py

```diff
@@ -965,18 +1093,47 @@ def plot_sensors(info, kind='topomap', ch_type=None, title=None,
         raise ValueError("ch_type must be one of %s not %s!" % (allowed_types,
                                                                 ch_type))
     picks = ch_indices[ch_type]
+    if len(picks) == 0:
+        raise ValueError('Could not find any channels of type %s.' % ch_type)
     if kind == 'topomap':
         pos = _auto_topomap_coords(info, picks, True)
     else:
         pos = np.asarray([ch['loc'][:3] for ch in info['chs']])[picks]
     def_colors = _handle_default('color')
     ch_names = np.array(info['ch_names'])[picks]
     bads = [idx for idx, name in enumerate(ch_names) if name in info['bads']]
-    colors = ['red' if i in bads else def_colors[channel_type(info, pick)]
-              for i, pick in enumerate(picks)]
+    if ch_groups is None:
+        colors = ['red' if i in bads else def_colors[channel_type(info, pick)]
+                  for i, pick in enumerate(picks)]
+    else:
+        if ch_groups == 'position':
+            colors = CH_GROUP_COLORS
+            ch_groups = _divide_to_regions(info, add_stim=False)
+            color_vals = [colors[key] for key in ch_groups.keys()]
+            ch_groups = list(ch_groups.values())
+        elif ch_groups == 'selection':
+            colors = CH_GROUP_COLORS
```

Comment:
Well I think all the dots in the same group should be the same color. Otherwise it would be impossible to distinguish the groups. Median color seems fine to me...